docs/cli.md

```diff
@@ -466,4 +466,7 @@ bigflow deploy \
 --dags-bucket europe-west1-12323a-bucket \
 --gcp-project-id my_gcp_dev_project \
 --clear-dags-folder
-```
\ No newline at end of file
+```
+
+## Scaffold project
+Use `bigflow start-project` command to create [example project](scaffold.md) and try all of the above commands by yourself.
```

Comment:
usa a `bigflow start-project` command to create a [sample project](scaffold.md) and try all of the above commands yourself.